### DISCUSSION TITLE
OCR 不能使用，一用就提示“发生未知错误”

### DISCUSSION THREAD

**Discussion by 190767486** [shuax/MouseInc.Settings#174] [Q&A]
OCR 不能使用，一用就提示“发生未知错误”， 是什么原因？我猜测是次数原因，另外我想请教一下是用的百度OCR还是华为OCR服务或者其他？ 如果我想换成百度OCR，在哪里调API Key这些？
![image](https://user-images.githubusercontent.com/92421340/137072498-d176357e-9ebd-41a6-a968-01778ac21784.png)

**Comment by andydznj** [comment]
的确啊，今天新出现的OCR不能使用的错误，不明原因，重新启动也不行

**Reply by andydznj** [reply]
请开发者帮帮忙，看看是什么原因

**Reply by hackerfei** [reply]
是的今天使用也发现了这个问题

**Comment by liangzj-gz** [comment]
![image](https://user-images.githubusercontent.com/92497258/137242494-91f75691-7ebb-4280-b734-1b0523a8eb25.png)
ORC用不了了

**Reply by kiomail** [reply]
I just can't use its OCR. Is the API OK?

**Comment by shuax** [comment]
![image](https://user-images.githubusercontent.com/6940583/137253758-c602060a-daf7-4dc9-97e3-ea49e3342ce1.png)

用的是有道的，我还能用，再观察下吧，实在不行我就换一个。

**Reply by kiomail** [reply]
It must be hard work. Congratulations.

**Reply by andydznj** [reply]
> ![image](https://user-images.githubusercontent.com/6940583/137253758-c602060a-daf7-4dc9-97e3-ea49e3342ce1.png)
> 
> 用的是有道的，我还能用，再观察下吧，实在不行我就换一个。

谢谢大佬啊，估计是有道发现太多，给设障碍了。再次感谢！！！！！！！！！！！！！！！！！！！！！！！！！

**Reply by ling-love-xin** [reply]
可否在设置上 开放 OCR的配置信息，这样我们也可以自定义下这个
![05711D44](https://user-images.githubusercontent.com/34054050/137438168-b2923c4c-9d41-4909-8b28-869b38bc3528.png)

**Reply by ealsm** [reply]
> ![image](https://user-images.githubusercontent.com/6940583/137253758-c602060a-daf7-4dc9-97e3-ea49e3342ce1.png)
> 
> 用的是有道的，我还能用，再观察下吧，实在不行我就换一个。

我的目前还是不能使用

**Comment by CheatMan-None** [comment]
现在是可以用了
![image](https://user-images.githubusercontent.com/75199690/137254570-96ce72cb-f02b-4700-87f2-b8b7c28d0560.png)

**Reply by kiomail** [reply]
It must be hard work. Congratulations.

**Reply by war525** [reply]
真的可以了，好几天了，都找到替代品了，还是这个好用

**Reply by kiomail** [reply]
> 真的可以了，好几天了，都找到替代品了，还是这个好用

You can't find a better one or even some one as good as it.

**Reply by andydznj** [reply]
感谢开发者修正问题
